docs/proposals/scheduling/policy-preemption/README.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Policy supports preemption by priority at runtime
+
+authors:
+- "@Poor12"
+
+creation-date: 2023-06-19
+
+---
+
+# Policy supports preemption by priority at runtime
+
+## Summary
+
+Currently, `PropagationPolicy` and `ClusterPropagationPolicy` already have a notion of priority. If multiple PropagationPolicies match the workload, Karmada will select the one with the highest priority. This is the case when the workload has not been selected by a policy. 
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: propagation-high-explicit-priority
+spec:
+  resourceSelectors:
+    - apiVersion: apps/v1
+      kind: Deployment
+      labelSelector:
+        matchLabels:
+          app: nginx
+  priority: 2
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+```
+
+This proposal propose a strategy which allows policies to support preemption by priority at runtime.
+Even if workloads have been propagated by a policy, they can be preempted by a high priority of policy.
+
+## Motivation
+
+Cluster administrators usually cannot foresee future expansion scenarios when configuring policies.
+They will usually start with a broad policy to set the base strategy. When an application requires special configuration,
+the administrator wants to provide a persionalized policy to take over the application.
+At this time, it hopes that the high-priority policy can preempt the low-priority policy.
+
+### Goals
+
+- Extend the API of PropagationPolicy/ClusterPropagationPolicy to allow high-priority policies to preempt low-priority policies.
+- Propose the implementation ideas for involved components, including `karmada-controller-manager` and `karmada-webhook`.
+- Prevent preemption caused by misconfiguration.
```

Comment:
> Prevent preemption caused by misconfiguration.

It would be great to give a more detail/clear description of this goal. 

I guess you want to say is that:
```
Propose a solution that is compatible with previous versions. The preemption capability will be disabled by default, upgrading the system from the previous version to the new one will not result in any changes in behavior and will not require any additional adaptation work.
```

Reply:
Updated.